frontend/src/app/components/ExperimentPlatforms/index.js

```diff
@@ -4,23 +4,20 @@ import { Localized } from "fluent-react/compat";
 
 import "./index.scss";
 
-const AVAILABLE_PLATFORMS = ["web", "addon", "mobile"];
+const AVAILABLE_PLATFORMS = ["web", "addon", "ios", "android"];
 
 export default function ExperimentPlatforms({ experiment }) {
-  const platforms = experiment.platforms.map(p => (p === "android" || p === "ios") ? "mobile" : p) || [];
-  const enabledPlatforms = AVAILABLE_PLATFORMS
-    .filter(platform => platforms.indexOf(platform) !== -1);
+  let enabledPlatforms = AVAILABLE_PLATFORMS
+    .filter(platform => experiment.platforms.indexOf(platform) !== -1);
```

Comment:
Easier to use `includes()` rather than `indexOf() !== -1`?

Reply:
c'mon man, i didn't even write that code :)